packages/core-modules/contracts/modules/ElectionModule.sol

```diff
@@ -36,8 +37,8 @@ contract ElectionModule is
         settings.minVotingPeriodDuration = 2 days;
         settings.minEpochDuration = 7 days;
         settings.maxDateAdjustmentTolerance = 7 days;
-        settings.nextEpochSeatCount = 3;
-        settings.minimumActiveMembers = 2;
+        settings.nextEpochSeatCount = uint8(firstCouncil.length);
+        settings.minimumActiveMembers = uint8(firstCouncil.length);
```

Comment:
I don't love this. Perhaps we need this to be a param on the init function too?

Reply:
Yup, agree, looks as hidden functionality if not

Reply:
Agree, I prefer it to be a param

Reply:
Ok, adding the constructor param. New PR to merge into this branch: https://github.com/Synthetixio/synthetix-v3/pull/848/files